codes/code_string.go

```diff
@@ -2,15 +2,15 @@
 
 package codes
 
-import "fmt"
+import "strconv"
 
 const _Code_name = "OKCanceledUnknownInvalidArgumentDeadlineExceededNotFoundAlreadyExistsPermissionDeniedResourceExhaustedFailedPreconditionAbortedOutOfRangeUnimplementedInternalUnavailableDataLossUnauthenticated"
 
 var _Code_index = [...]uint8{0, 2, 10, 17, 32, 48, 56, 69, 85, 102, 120, 127, 137, 150, 158, 169, 177, 192}
 
 func (i Code) String() string {
 	if i >= Code(len(_Code_index)-1) {
-		return fmt.Sprintf("Code(%d)", i)
+		return "Code(" + strconv.FormatInt(int64(i), 10) + ")"
```

Comment:
`strconv.Itoa(i)` ?

Reply:
This is generated code; we aren't supposed to even look at it.  :)

Reply:
Oh! Did realize that.